knowledge_base/Database-Optimization.md

```diff
@@ -1,24 +1,37 @@
 # Query Optimization
```

Comment:
Honestly, I would delete this entire file. It covers very rudimentary optimizations that anyone working with the DB should know by heart. We are building much better processes now that include looking at query plans and load testing. This information provides little value and I highly doubt anyone has or will reference it. Optimizations happen on a case-by-case basis by thoroughly reviewing the query plan.